index.html

```diff
@@ -1,21 +1,520 @@
+            activities in your account.
+          </p>
+        </div>
+      </div>
+      <div
+        class="
+          flex flex-col
+          justify-center
+          items-center
+          laptop-large:px-4
+          md:px-2
+          py-5
+        "
+      >
+        <img
+          src="./styles/assets/home/desktop/icon-banking-and-coverage.svg"
+          alt=""
+        />
+        <p
+          class="heading-3-small pt-8 text-center text-secondary-san-juan-blue"
+        >
+          Banking & Coverage
+        </p>
+        <p
+          class="
+            body-copy-1
+            pt-4
+            text-center text-secondary-light-san-juan-blue
+          "
+        >
+          With our platform, you can speed up account onboarding and support
+          ongoing payments for checking, savings, credit card, and brokerage
+          accounts.
+        </p>
+      </div>
+      <div
+        class="flex flex-col items-center py-10 md:px-1 laptop-large:px-4 pt-12"
+      >
+        <img
+          src="./styles/assets/home/desktop/icon-consumer-payments.svg"
+          alt=""
+        />
+        <p
+          class="heading-3-small pt-8 text-center text-secondary-san-juan-blue"
+        >
+          Consumer Payments
+        </p>
+        <p
+          class="
+            body-copy-1
+            pt-4
+            text-center text-secondary-light-san-juan-blue
+          "
+        >
+          It’s easier to set up secure bank payments with us through a flow
+          designed with the user experience in mind. Customers could instantly
+          authenticate their account.
+        </p>
+      </div>
+    </div>
+    <div
+      class="
+        flex flex-col
+        laptop-large:flex-row laptop-large:justify-between
+        justify-center
+        laptop-large:px-40
+        items-center
+        px-6
+        pb-20
+      "
+    >
+      <p
+        class="
+          heading-3-big
+          md:text-5xl md:leading-14
+          laptop-large:w-1/2
+          text-secondary-san-juan-blue
+        "
+      >
+        Ready to start?
+      </p>
+      <div
+        class="
+          md:flex md:flex-row md:relative md:items-end md:justify-end md:px-36
+          laptop-large:px-0 laptop-large:pl-56
+          w-full
+        "
+      >
+        <input
+          class="
+            body-copy-1
+            w-full
+            pt-4
+            pr-6.125
+            pb-3.5
+            pl-6.625
+            mt-6
+            rounded-3xl
+            bg-white
+          "
+          type="email"
+          placeholder="Enter email address"
+        />
+        <button
+          class="
+            btn btn--primary
+            md:absolute
+            mt-4
+            md:mt-0 md:pt-5 md:pb-5
+            w-full
+            md:w-max
+            text-center
+          "
+        >
+          Schedule a Demo
+        </button>
+      </div>
+    </div>
+    <div
```

Comment:
This section is a footer and should use the `footer` element. The logo is not inline with the rest of the nav links
<img width="397" alt="Screen Shot 2021-11-22 at 8 16 50 PM" src="https://user-images.githubusercontent.com/26441464/142958207-179895bf-32bd-4a59-8bb7-e2f325abd0c9.png">

Reply:
footer nav properly centered.